### DISCUSSION TITLE
Support for multiple GPUs on local executor

### DISCUSSION THREAD

**Discussion by petroslk** [nextflow-io/nextflow#4449] [Q&A]
Hello! 

Is there a plan to integrate GPU scheduling into the local executor? We need to use nextflow on a large workstation with multiple GPUs. There was a solution proposed at #997  but I am not sure this is on your radat at the moment. I guess something along the lines of using  `CUDA_VISIBLE_DEVICES` and `nvidia-smi` to track what each GPU is running would do the trick? Is there any workaround available at the moment?

Thanks

**Comment by andreas-wilm** [comment]
For reference see also discussions here: https://community.seqera.io/t/proper-way-to-utilize-1-gpus-in-a-single-machine/455/7
